environments/production/terraform.yml

```diff
@@ -190,10 +190,11 @@ servers:
     volume_size: 150
 
   - server_name: "pillow0-production"
-    server_instance_type: r5.2xlarge
+    server_instance_type: r5a.4xlarge
```

Comment:
why was r5 changed to r5a ? The difference I can see is that there are no ECU and its slightly cheaper. Was there a reason we weren't using r5a.2xlarge before?

Reply:
I think I just hadn't originally noticed r5a the first time around. What I wanted to do was just increase memory in the cheapest way possible without necessarily increasing CPU. It looked like r5a just doesn't make the same guarantees about CPU and that seemed fine.

I agree that this is worth revisiting at some point. In steady state we may not need such a large machine, or we may be able to use that capacity better to get higher throughput by tweaking the number of processes and other tuning.